README.md

```diff
@@ -1,250 +1,326 @@
+``` 
+zypper install --type pattern devel_basis zypper install git ncurses-devel \
+zlib-devel gawk time unzip perl-libxml-perl flex wget gawk gettext-runtime \
+quilt python libopenssl-devel 
 ```
-zypper install --type pattern devel_basis
-zypper install git ncurses-devel zlib-devel gawk time \
-  unzip perl-libxml-perl flex wget gawk gettext-runtime quilt python libopenssl-devel
-```
-On Arch/Antergos:
+On Arch/Antergos: 
 ```
 pacman -S base-devel git ncurses lib32-zlib gawk time unzip perl-xml-libxml \
- flex wget gettext quilt python2 openssl
+flex wget gettext quilt python2 openssl 
 ```
 
-### Building all firmwares
 
-To get the source and build the firmware locally use:
+### Build-System-Structure  
+
+Where can I change something? What do these files do? 
+
+``` 
+:: Settings
+
+config.mk         - select what package lists to build
+configs/          - 
+modules           - define git repository, revision, etc of OpenWRT and Packages
+
+patches/          - Patches against the OpenWRT directory  
+packagelists/     - Package Lists  
+profiles/         - List of router profiles for each architecture - names come 
+                      from the Imagebuilder
+embedded-files/   - Files that will be included in the images without further
+                      processing. Its mirrored to / . Thus, you must provide a
+                      directory tree there, to put your file at the right point
+
+:: Building 
+
+Makefile                - Does all the stuff. Cloning and Updating OpenWRT,
+                            patching, running make 
+assemble_firmware.sh    - Script for running the Imagebuilder for a target
+
+
+:: Images  
+
+firmwares/            - Folder where the final images end up 
 
 ```
-git clone https://github.com/freifunk-berlin/firmware.git
-cd firmware
+
+
+### Building all firmware images
+
+To get the source and build the firmware locally use:
+
+``` 
+git clone https://github.com/freifunk-berlin/firmware.git 
+cd firmware 
 make
 ```
 
-The build will take some time. You can improve the build time with [build options](https://openwrt.org/docs/guide-developer/build-system/use-buildsystem)
+The build will take some time. You can improve the build time with [build
+options](https://openwrt.org/docs/guide-developer/build-system/use-buildsystem)
 such as `-j <number of cores>`. `V=s` will give more verbose error messages.
 
 An internet connection is required during the build process. A good internet
 connection can improve the build time.
 
 You need approximately 10GB of space for the build.
 
-### Directory Layout
-
-You can find the actual firmware images generated by the ImageBuilder (and the ImageBuilder itself)
-in `firmwares`. The layout looks like the following:
 
+### Directory Layout of `firmwares/`
+
+You can find the actual firmware images generated by the ImageBuilder (and the
+ImageBuilder itself) in `firmwares/`. The layout looks like the following:
+
+``` 
+firmwares/ 
+  TARGET/ 
+    backbone/ 
```

Comment:
Should we use the changed packagelists here (notunnel, manual, ...)?

Reply:
Adjusted to new naming.